prow/plugins/trigger/ic_test.go

```diff
@@ -174,14 +288,14 @@ func TestHandleIssueComment(t *testing.T) {
 					Name:      "job",
 					AlwaysRun: true,
 					Context:   "pull-job",
-					Trigger:   "@k8s-bot test this",
+					Trigger:   `(?m)^/test all,?(\s+|$)`,
```

Comment:
yup! i think this is the best explanation regarding choosing betwee test coverage and churn reduction. to simplify, i will only test regex `^/test all$` (without comma, multiple line, and extra spaces behind).